File: app/src/main/java/modernjava/ex03.java
package modernjava;

import java.util.Arrays;
import java.util.List;

public class ex03 {
    public static void main(String[] args) {
        List<Integer> values = Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);

        List<Integer> result = values.stream()
                .filter(v -> v % 2 == 0)
                .map(v -> v * v)
                .toList();

        System.out.println(result);
    }
}

File: app-kotlin/src/main/kotlin/modernjava/ex03.kt
package modernjava

class ex03 {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val values = listOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
            val result = values.filter { it % 2 == 0 }.map { it * it }
            println(result)
        }
    }
}
